programmers/기능개발.py

```diff
@@ -0,0 +1,21 @@
+
+import math
+
+def solution(progresses, speeds):
+    answer = []
+    items = []
+    for i in range(len(progresses)):
+        item = (100 - progresses[i]) / speeds[i]
+        items.append(math.ceil(item))
+    std = items.pop(0)
+    count = 1
+    while items:
+        item = items.pop(0)
+        if std < item:
+            std = item
+            answer.append(count)
+            count = 1
+        else:
+            count += 1
+    answer.append(count)
+    return answer
```

Comment:
```suggestion
    return answer
    
```

파일의 마지막은 항상 한칸 띄워주세요 :)

Reference
- [파일 마지막에 빈 줄을 넣는 이유](https://velog.io/@jesop/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EC%BD%94%EB%93%9C-%EB%A7%88%EC%A7%80%EB%A7%89%EC%97%90-%EB%B9%88-%EC%A4%84-%EC%82%BD%EC%9E%85%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0)